portal-ui/src/screens/Console/Support/Register.tsx

```diff
@@ -913,46 +943,7 @@ const Register = ({ classes }: IRegister) => {
               />
             </Box>
           </Box>
-
-          <Box
-            sx={{
-              border: "1px solid #eaeaea",
-              borderRadius: "2px",
-              display: "flex",
-              padding: "23px",
-              marginTop: "40px",
-              alignItems: "center",
-              justifyContent: "center",
-            }}
-          >
-            <Box
-              sx={{
-                display: "flex",
-                alignItems: "center",
-              }}
-            >
-              <OfflineRegistrationBackIcon /> Cluster does not have internet
-              access? Use{" "}
-            </Box>
-            <Box
-              sx={{
-                display: "flex",
-                alignItems: "center",
-                marginLeft: "8px",
-              }}
-            >
-              <Link
-                className={classes.link}
-                onClick={() => {
-                  fetchSubnetRegToken();
-                  setOnlineActivation(!onlineActivation);
-                }}
-              >
-                Offline Activation.
-              </Link>
-            </Box>
-          </Box>
-        </React.Fragment>
+        </Fragment>
```

Comment:
This field is always disabled (at 912)
```suggestion
        {displaySubnetProxy && (
                  <InputBoxWrapper
                    disabled
                    id="subnetProxy"
                    name="subnetProxy"
                    placeholder=""
                    onChange={() => {}}
                    label=""
                    value={proxyConfigurationCommand}
                    overlayIcon={<CopyIcon />}
                    extraInputProps={{
                      readOnly: true,
                    }}
                    overlayAction={() =>
                      navigator.clipboard.writeText(proxyConfigurationCommand)
                    }
                  />
                )}
```

Reply:
That's the expected behavior, we can only copy the text inside the input